src/transformers/models/qwen2_vl/processing_qwen2_vl.py

```diff
@@ -57,7 +57,7 @@ class Qwen2VLProcessor(ProcessorMixin):
 
     attributes = ["image_processor", "tokenizer"]
     valid_kwargs = ["chat_template"]
-    image_processor_class = "Qwen2VLImageProcessor"
+    image_processor_class = ("Qwen2VLImageProcessor", "Qwen2VLImageProcessorFast")
```

Comment:
Indeed didn't catch that thanks! Best thing would be to have this for now: 
```suggestion
    image_processor_class = "AutoImageProcessor"
```
Otherwise, the fast image processor will be loaded by default when not specifying `use_fast`, even if the original checkpoint was saved with a slow processor, which is not the behavior we want for now, although it will be in the future when we make fast image processors the default.